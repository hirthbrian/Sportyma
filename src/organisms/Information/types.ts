export interface Props {
  info: string;
}
