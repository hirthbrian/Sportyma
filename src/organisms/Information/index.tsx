import { SectionTitle, KanitText } from "../../atoms";

import { Props } from "./types";

const Information = ({ info }: Props) => {
  return (
    <>
      <SectionTitle title="Informations" />
      <KanitText weight={300} color="#152D4B">
        {info}
      </KanitText>
    </>
  );
};

export default Information;
