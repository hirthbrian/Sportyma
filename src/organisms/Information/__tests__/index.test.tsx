import React from "react";
import renderer from "react-test-renderer";

import Information from "..";

describe("<Information />", () => {
  let props;

  beforeEach(() => {
    props = {
      info: "bla bla bla",
    };
  });

  it("renders correcly", () => {
    expect(renderer.create(<Information {...props} />).toJSON()).toBeTruthy();
  });
  it("match snapshot", () => {
    const tree = renderer.create(<Information {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
