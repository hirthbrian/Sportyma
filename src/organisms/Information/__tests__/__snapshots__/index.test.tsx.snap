// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Information /> match snapshot 1`] = `
Array [
  <Text
    size={20}
    style={
      Array [
        Object {
          "color": "#152D4B",
          "fontFamily": "Kanit-Medium",
          "fontSize": 20,
        },
      ]
    }
    weight={500}
  >
    Informations
  </Text>,
  <Text
    color="#152D4B"
    style={
      Array [
        Object {
          "color": "#152D4B",
          "fontFamily": "Kanit-Light",
          "fontSize": 16,
        },
      ]
    }
    weight={300}
  >
    bla bla bla
  </Text>,
]
`;
